src/indexing/EventIndexPeg.js

```diff
@@ -66,8 +68,25 @@ class EventIndexPeg {
      */
     async initEventIndex() {
```

Comment:
only place I see this getting called is the EventIndexPanel?

Reply:
And from the `init()` method of the `EventIndexPeg` which gets called in `Lifecycle.js`. Something wrong with that?

Reply:
ah perfect, missed it flicking through